packages/material/scss/scrollview/_layout.scss

```diff
@@ -1 +1,11 @@
 @import "~@progress/kendo-theme-default/scss/scrollview/layout";
+
+@include exports('scrollview/layout') {
```

Comment:
The same module is defined in `~@progress/kendo-theme-default/scss/scrollview/layout`. This means the code in this block is never included, because the names are the same.